### PR TITLE
Create switchstatements

### PR DESCRIPTION
**Pull request by Monica-Raeyln**
I have made the 3 switch statements, hopefully this is what I was supposed to do and everything looks alright. If you need anything additional let me know. I am generally not available during the day until 4-5 pm MST so II try to get this stuff done at night.

### CODE REVIEW

File: switchstatements [hunk 1]
```diff
@@ -0,0 +1,55 @@
...
+
+switch (randomActivity) {
+  case 0:
+  activity = 'wrote a program';
+  break;
+  case 1:
+  activity = 'debugged a program';
+  break;
+  case 2:
+  eightBall = 'tested and evaluated a new program';
```
Comment (TracyAWh) [inline switchstatements:28]: // How do I make changes to the code? For example, line 28:
activity = 'tested and evaluated a new program';

File: switchstatements [hunk 2]
```diff
@@ -0,0 +1,55 @@
...
+  break;
+  case 3:
+  eightBall = 'identified areas to modify a program';
```
Comment (TracyAWh) [inline switchstatements:31]: activity = 'identified areas to modify a program';

File: switchstatements [hunk 3]
```diff
@@ -0,0 +1,55 @@
...
+  break;
+  case 4:
+  eightBall = 'got lost down a rabbit hole trying to create a new program';
```
Comment (TracyAWh) [inline switchstatements:34]: activity = 'got lost down a rabbit hole trying to create a new program';

Comment (TracyAWh) [reply switchstatements:34]: // PS I love this line!

**Comment by Monica-Raeyln** [comment]
I will fix those. Apparently I overlooked them. It was late and I was tired sorry about that I’ll log in and fix them ASAP!


Sent from Yahoo Mail for iPhone


On Thursday, May 27, 2021, 1:45 PM, TracyAWh ***@***.***> wrote:




@TracyAWh commented on this pull request.

In switchstatements:
> +
+switch (randomActivity) {
+  case 0:
+  activity = 'wrote a program';
+  break;
+  case 1:
+  activity = 'debugged a program';
+  break;
+  case 2:
+  eightBall = 'tested and evaluated a new program';
+  break;
+  case 3:
+  eightBall = 'identified areas to modify a program';
+  break;
+  case 4:
+  eightBall = 'got lost down a rabbit hole trying to create a new program';

// PS I love this line!

**Comment by TracyAWh** [comment]
Cool. Just typos. Now I’ve got to do my job.

I got lost trying to figure out how to create a branch to make my changes to Monica’s code  though. Any one remember how to do that?

**Comment by von-amari** [comment]
Did you get it figured out, Tracy? I put a link in our group discussion that may help. Once you do the git remote setup, you should be able to do git pull to grab what Monica did.

Sent from my iPhone

On May 27, 2021, at 1:48 PM, TracyAWh ***@***.***> wrote:

﻿

Cool. Just typos. Now I’ve got to do my job.

I got lost trying to figure out how to create a branch to make my changes to Monica’s code though. Any one remember how to do that?

**Pull request merged**